static/bidder-info/copper6ssp.yaml

```diff
@@ -0,0 +1,21 @@
+endpoint: "https://endpoint.copper6.com/"
+maintainer:
+  email: "info@copper6.com"
+capabilities:
+  site:
+    mediaTypes:
+    - banner
+    - video
+    - native
+  app:
+    mediaTypes:
+    - banner
+    - video
+    - native
+userSync:
+  redirect:
+    url: "https://csync.copper6.com/pbserver?gdpr={{.GDPR}}&gdpr_consent={{.GDPRConsent}}&ccpa={{.USPrivacy}}&gpp={{.GPP}}&gpp_sid={{.GPPSID}}&redir={{.RedirectURL}}"
+    userMacro: "[UID]"
+  iframe:
+    url: "https://csync.copper6.com/pbserverIframe?gdpr={{.GDPR}}&gdpr_consent={{.GDPRConsent}}&ccpa={{.USPrivacy}}&gpp={{.GPP}}&gpp_sid={{.GPPSID}}&pbserverUrl={{.RedirectURL}}"
+    userMacro: "[UID]"
```

Comment:
The iframe support doesn't appear to be working. I'm getting a 404 using the following: `https://csync.copper6.com/pbserverIframe?gdpr=0&gdpr_consent=CPuESnSPuESnSPiAAAENCZCAAIAAAAAAAAAAAQAAAAAEAAAAAA&ccpa=&gpp=&gpp_sid=&pbserverUrl=http%3A%2F%2Flocalhost%3A8000%2Fsetuid%3Fbidder%3Dcopper6ssp%26gdpr%3D0%26gdpr_consent%3DCPuESnSPuESnSPiAAAENCZCAAIAAAAAAAAAAAQAAAAAEAAAAAA%26gpp%3D%26gpp_sid%3D%26f%3Db%26uid%3DabcUID`

<img width="1232" alt="Screenshot 2024-08-02 at 12 57 33 PM" src="https://github.com/user-attachments/assets/0519905d-201f-46d4-8224-1338b18ad0f9">

Reply:
fixed